Libraries/Components/Pressable/Pressable.js

```diff
@@ -27,6 +27,11 @@ import type {
   LayoutEvent,
   MouseEvent,
   PressEvent,
+  // [TODO(macOS GH#774)
+  FocusEvent,
+  BlurEvent,
+  KeyEvent,
+  // ]TODO(macOS GH#774)
 } from '../../Types/CoreEventTypes';
```

Comment:
These changes in `Pressable` look like a difference between `main` and `0.66-stable`, and not part of your change? They should go away if you target main again, but also tell me I need to go update 0.66-stable with these changes.

Reply:
Afaik the main branch should be (at least) rn-0.66, so lets target the main branch again ;-)
I am curious why this did not create an issue already, don't you have subclasses of RCTView?

Reply:
Yeah, both `main` and `0.66-stable` are about equal at the moment. 
It looks like the RCTEventDispatcher/Protocol mismatch has been causing a warning, but not an error. I wonder why we also have it, given RN Core doesn't have an `initWithEventDispatcher` on `RCTView`. My guess is it's because our `RCTView` has focus/blur events..